extra/mariabackup/xtrabackup.cc

```diff
@@ -1745,11 +1754,13 @@ static int prepare_export()
       " --defaults-file=./backup-my.cnf --defaults-group-suffix=%s --datadir=."
       " --innodb --innodb-fast-shutdown=0 --loose-partition"
       " --innodb_purge_rseg_truncate_frequency=1 --innodb-buffer-pool-size=%llu"
-      " --console  --log-error= --skip-log-bin --bootstrap  < "
+      " --console --log-error= --skip-log-bin --bootstrap"
+      " --innodb-force-recovery=%lu < "
       BOOTSTRAP_FILENAME IF_WIN("\"",""),
       mariabackup_exe,
       (my_defaults_group_suffix?my_defaults_group_suffix:""),
-      xtrabackup_use_memory);
+      xtrabackup_use_memory,
+      srv_force_recovery);
```

Comment:
I think that we should output this special option only when it is set. Maybe, just add `"%s`" to the format string and pass `srv_force_recovery ? " --innodb-force-recovery=1" : ""` as the parameter.